monitoring/templates/lambda/notify-slack-batch.js

```diff
@@ -0,0 +1,99 @@
+var https = require("https");
+var util = require("util");
+
+exports.handler = function(event, context) {
+    console.log(JSON.stringify(event, null, 2));
+    const londonTime = new Date().toLocaleString("en-GB", {timeZone: "Europe/London"});
+    const now = new Date(londonTime).getHours();
+    if (now >= +"${quiet_period_start_hour}" && now < +"${quiet_period_end_hour}") {
+        console.log("In quiet period, dismissing alarm");
+        return;
+    }
+
+    const eventMessage = JSON.parse(event.Records[0].Sns.Message);
+    console.log(JSON.stringify(eventMessage.detail, null, 2));
+    
+    let environment_name = "unknown";
+    let sendSlackNotification = false;
+    let severity = "ok";
+    let icon_emoji = ":question:";
+    
+    switch(eventMessage.detail.status) {
+      case "SUCCEEDED":
+        severity = "ok";
+        icon_emoji = ":yep:";
+        sendSlackNotification = true;
+        break;
+      case "FAILED":
+        severity = "critical";
+        icon_emoji = ":siren:";
+        sendSlackNotification = true;
+        break;
+      default:
+        severity = "ok";
+        icon_emoji = ":yep:";
+        sendSlackNotification = false;
+    }
+    
+    console.log("severity:" + severity);
+    console.log("icon_emoji:" + icon_emoji);
+    console.log("sendSlackNotification:" + sendSlackNotification);
+    
+    // let jobId = eventMessage.detail.jobId;
+    // let jobName = eventMessage.detail.jobName;
+    // let jobQueue = eventMessage.detail.jobQueue;
+    let logsPath = "https://eu-west-2.console.aws.amazon.com/cloudwatch/home?region=eu-west-2#logsV2:log-groups/log-group/$252Faws$252Flambda$252F" + environment_name + "-notify-delius-core-slack-channel-batch";
```

Comment:
Can we get some of these vars from terraform, as this .js file is templated in [data.tf](https://github.com/ministryofjustice/hmpps-delius-core-terraform/pull/375/files#diff-1a4d37934355f9821b17b6af88e2d578R97-R105)?
```suggestion
    let logsPath = "https://eu-west-2.console.aws.amazon.com/cloudwatch/home?region=eu-west-2#logsV2:log-groups/log-group/$252Faws$252Flambda$252F${lambda_name}";
```